docs/source/tutorial/visualise_pipeline.md

```diff
@@ -184,7 +197,7 @@ def create_pipeline(**kwargs) -> Pipeline:
         [
             node(
                 func=compare_shuttle_speed,
-                inputs="shuttle_speed_data",
+                inputs=None,
```

Comment:
My only point, why not do a simple chart from the real data, `input=None` is a bit weird

Reply:
Yes fair point. I'll try and get something working!

Reply:
[I have something here](https://github.com/datajoely/modular-spaceflights/tree/main/src/modular_spaceflights/pipelines/reporting)

Reply:
ah saw this too late and implemented something basic. Let me know if it makes sense, otherwise I'm happy to try out your example.